Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -940,14 +1015,15 @@ def get_random_tests(tests_num, conf=None, id_set=None, server_version='0'):
     return tests
 
 
-def get_test_list(files_string, branch_name, two_before_ga_ver='0', conf=None, id_set=None):
+def get_test_list_and_content_packs_to_install(files_string, branch_name, two_before_ga_ver='0', conf=None, id_set=None):
     """Create a test list that should run"""
     (modified_files, modified_tests_list, changed_common, is_conf_json, sample_tests, is_reputations_json,
      is_indicator_json) = get_modified_files(files_string)
 
     tests = set([])
+    packs_to_install = set([])
     if modified_files:
-        tests = find_tests_for_modified_files(modified_files, conf, id_set)
+        tests, packs_to_install = find_tests_and_content_packs_for_modified_files(modified_files, conf, id_set)
```

Comment:
verify that you find packs, like for tests.

Reply:
why is that needed?

Reply:
to make sure that you really found packs, otherwise raise a proper error

Reply:
but why would we want to raise an error if no packs were found? it can happen that no packs were modified..